### ISSUE TITLE
Layout issue on wide screens

### ISSUE THREAD

**Issue by BostNisReport** [wgoldm/plugin-notes-fork#3]
On a narrow screen the edit box looks like this:
![1](https://user-images.githubusercontent.com/16148220/40026386-d4210496-57dd-11e8-853d-6dd733d7eb92.jpg)
however it should look like this:
![2](https://user-images.githubusercontent.com/16148220/40026411-e8fad0b8-57dd-11e8-902b-db607ade6dfc.jpg)
I guess the solution is to make the edit box full-width within the box it resides please.

**Comment by tmonzur** [comment]
Updated as per screen.

**Comment by tmonzur** [comment]
![layout issue updated](https://user-images.githubusercontent.com/38818306/40322026-d405c634-5d52-11e8-9280-ec790255372d.png)

**Comment by BostNisReport** [comment]
Yes, this works, but please can we have a better layout like this one?

![1](https://user-images.githubusercontent.com/16148220/40326534-3caf37e8-5d48-11e8-9702-9ea01b6186dd.jpg)

**Comment by tmonzur** [comment]
Updated...
